cli/args/mod.rs

```diff
@@ -1255,11 +1261,29 @@ impl CliOptions {
     }
   }
 
-  pub fn node_modules_dir_enablement(&self) -> Option<bool> {
-    self
-      .flags
-      .node_modules_dir
-      .or_else(|| self.workspace().node_modules_dir())
+  pub fn node_modules_mode(&self) -> Result<Option<NodeModulesMode>, AnyError> {
+    if *DENO_FUTURE {
+      if let Some(flag) = self.flags.node_modules_mode {
+        return Ok(Some(flag));
+      }
+      self.workspace().node_modules_mode().map_err(Into::into)
+    } else {
+      Ok(
+        self
+          .flags
+          .node_modules_dir
```

Comment:
Maybe we should just support `nodeModules` with `DENO_FUTURE=0` too? Right now it only looks at `nodeModulesDir`

Reply:
Meh, we're going to get rid of it in a bit.